### DISCUSSION TITLE
#51 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by Emilyxiannv** [tog-ETH-er/togETHer#34] [General]
Agenda:

-Community updates
-The fifth rotating chairman nomination
-Fireside discussion

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/s/LxNajKWJw4RV

会议 ID：891 5098 6775

**Comment by Riely007** [comment]
### 以太坊翻译小组

- 官网翻译 & review 进行中
- smart contract security 新增

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

8个PR #863 ～ #871

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了5场分享会

- 《DEV with FOUNDRY  ( ETHKL1 & Dapp-Learning  ) 》 by Alan
  - https://www.youtube.com/watch?v=Jeiul1Vyorg

- 《Arweave and storage-based consensus paradiam 》 by Xiong Wei
  - https://www.youtube.com/watch?v=_D5-FUcBcwE
 
- 《W3C DID的基本概念及应用》by w3tester
  - https://www.youtube.com/watch?v=Kfc0NIffAms

- 《Introducing SafeStake ( ETH2 0 stake ) 》by ethstak3r
  - https://www.youtube.com/watch?v=nxPMH7GS_WE
 
- 《当年的加密主义思潮-《True nyms and crypto anarchy》读书会》by arc
  - https://www.youtube.com/watch?v=V6YTfJKELZs

### rewards

DLIP-023 sharing meetup reward

https://github.com/Dapp-Learning-DAO/DLIP/issues/23

**Comment by Thegaram** [comment]
### Crypto Tech Night

11.03: CTN #18 做市商与安全性（过几天可以在这里看回放：https://space.bilibili.com/1560365248 ）

![image](https://user-images.githubusercontent.com/7571518/201665658-68e49d7d-f2c5-42ca-a056-998274cb41d9.png)

Next up: 12月CTN #19，主题未定

未来的新想法：跟Scroll一起做一个以zk dapp为主的event series，麻烦各位有空推荐一些zkp dapp

**Comment by stephanieeee-ho** [comment]
### ECN
- 常规翻译和周报
- 与 orbiter finance 的一期podcast，这周五上线

**Comment by Stamp9** [comment]
### 原语里弄

新课题池流程测试

**Comment by xrdavies** [comment]
### Rebase 社区

- 14 期 Web3 日报
- 若干招聘信息和活动
- 确定 2022Q4 Rebase Hackathon 方案

**Comment by Emilyxiannv** [comment]
### DAOSquare

DAOSquare 构建的孵化器产品Phoenix Beta V1.0测试版上线：https://beta.phoenix.fi/home （Goerli 测试网）

**Comment by bobjiang** [comment]
# GitcoinCN

- 周报更新
- GR16的最新消息：https://gov.gitcoin.co/t/discussion-feedback-request-grants-protocol-alpha-round-eligibility/11873

**Comment by 0xPabloLI** [comment]
### 普朗克社区

- 11.12 举办小V共读会&北邮链协 -- The Revenue-Evil Curve: https://plancker.notion.site/13-V-3-53bd88b9f2404ed2b184d650e15a94a3
- 探讨调研EIP-4337周边生态方向 https://plancker.notion.site/idea-f6accb182af1474fbf2c506fe63e4c53
